1-resilient-messaging/src/Application/OrderService.php

```diff
@@ -7,16 +7,27 @@
 use App\Domain\Order;
 use App\Domain\OrderRepository;
 use App\Domain\ShippingService;
+use App\Event\OrderWasPlaced;
+use Ecotone\Messaging\Attribute\Asynchronous;
```

Comment:
Czy sądzisz, że powiniśmy mieć to już zaimportowane w klasie, tylko nie użyte?
W ten sposób nie trzeba byłoby importować, byłoby łatwiej w przypadku, gdy ktoś nie korzysta z IDE do PHP